examples/data/quiz_multichoice_answers.csv

```diff
@@ -0,0 +1,34 @@
+id,question_id,text,grade,feedback
+0,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>5</p>,0.0,nope
+1,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>6</p>,0.0,nah
+2,1a7b77c1-e9e1-4f22-ba84-56a74b733419,<p>11</p>,1.0,yes
```

Comment:
I hit this while trying to use the example files to run your enclave code in [this PR](https://github.com/openstax/raise-moodle-spikes/pull/73), but it might be good to make this consistent with actual content where (I believe) all of the multichoice questions always have four options. Currently in this test file it looks like there's a range of 2, 3, or 4 options for different questions which might create unnecessary confusion for users when writing / testing their code.

Reply:
Dang, I'd noticed this too and tried to correct for this and I think in my git dilemma this fix got lost. I'll make that change now.